__tests__/app.test.js

```diff
@@ -98,3 +88,34 @@ describe("GET: 200 /api/articles/:article_id", () => {
     });
   });
 });
+
+describe("GET: 200 /api/articles", () => {
+  test("Should return the articles array sorted in descending order and without a body property", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body.articles;
+        expect(articles).toHaveLength(13);
+        const articlesSortedByDate = articles.sort((a, b) =>
+          a.created_at.localeCompare(b.created_at)
+        );
+        expect(articlesSortedByDate).toEqual(articles);
+        articles.forEach((article) => {
+          expect(article).not.toContain(article.body);
+          expect(typeof article.comment_count).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+        });
+      });
+  });
+  describe("GET: 400 /api/articles", () => {
+    test("Should return a message saying the URL endpoint is invalid", () => {
+      return request(app)
+        .get("/api/wrong_endpoint")
```

Comment:
This should only be tested once through your app. A spelling mistake would not match any endpoint so isn't particular to an individual endpoint.